### ISSUE TITLE
TLSCipherSuite file not recognised by config

### ISSUE THREAD

**Issue by Synchro** [jedisct1/pure-ftpd#45]
When I have a file in `/etc/pure-ftpd/conf/TLSCipherSuite` containing the suites I want (the built-in ones with `DES` removed), pure-ftpd complains on startup:

> Restarting ftp server: /usr/sbin/pure-ftpd-wrapper: Invalid configuration file /etc/pure-ftpd/conf/TLSCipherSuite: No corresponding directive

I can see the mapping of `TLSCipherSuite` to `-J` in the config parser so I'm not sure why this is happening. Any idea?

This file was working in 1.0.36; this error started appearing when I updated to 1.0.43, running on Ubuntu trusty.

**Comment by jedisct1** [comment]
The `No corresponding directive` message doesn't exist anywhere in pure-ftpd.

Maybe this option is not available in the Debian-specific startup script, or doesn't work?

**Comment by jedisct1** [comment]
Also, the default cipher suite doesn't include DES, so you usually don't need to change it.

**Comment by racke** [comment]
Right, this message is from the wrapper script - but it should recognize this directive. I just tested it with my Debian install. Which exact package version do you use on Ubuntu?

**Comment by Synchro** [comment]
I found that the `-J` option was missing from the wrapper script in my 1.0.43 package - I added it in manually and it's happy now. It may just be a problem with the PPA I got this version from - the stock repos only have 1.0.36.  I'll try to find where this is happening with the maintainer if I can.

I checked your changelog - 3DES was removed in 1.0.44, one version after what I'm running.

All accounted for now, thanks!
